taxi/forms.py

```diff
@@ -0,0 +1,58 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+from django import forms
+
+from taxi.models import Driver, Car
+
+
+def validate_license_number(lic_number: str) -> str:
+    if len(lic_number) != 8:
+        raise ValidationError("License number must be exactly "
+                              "8 characters long")
+    elif len(lic_number) == 8:
+        for i in range(0, 3):
+            if lic_number[i] != lic_number[i].capitalize():
+                raise ValidationError("The first 3 letters "
+                                      "must be capitalized")
+            if not lic_number[i].isalpha():
+                raise ValidationError("The first 3 characters must be letters")
+        for elem in range(3, 8):
+            if not lic_number[elem].isnumeric():
+                raise ValidationError("The last 5 characters must be digits")
+    return lic_number
+
+
+class DriverCreationForm(UserCreationForm):
+
+    class Meta(UserCreationForm.Meta):
+        model = Driver
```

Comment:
Consider using `get_user_model()` instead of directly referencing the `Driver` model. This will make your code more flexible and aligned with Django best practices.